src/lib/statsv2.svelte

```diff
@@ -0,0 +1,168 @@
+<!--
+  statsv2.svelte:
+    Stats highlighting the value of fixNation's work-v2
+-->
+<script lang="ts">
+  import { base } from '$app/paths';
+  let sections = [
+    {
+      number: '#1',
+      subtitle: 'largest clinic for homeless cats',
+      picture: 'statsv2/statImage1.png',
+      alt: 'Sedated cats'
+    },
+    {
+      number: '220k',
+      subtitle: 'total cats spayed or neutered',
+      picture: 'statsv2/statImage2.png',
+      alt: 'Cat on rug'
+    },
+    {
+      number: '21%',
+      subtitle: 'of cats abandoned because their of their residency',
+      picture: 'statsv2/statImage3.png',
+      alt: 'Momma cat with her kittens'
+    },
+    {
+      number: '12k',
+      subtitle: 'TNR volenteers trained',
+      picture: 'statsv2/statImage4.png',
+      alt: 'FixNation staff holding a cat'
+    },
+    {
+      number: '89%',
+      subtitle: 'profits go directly to FixNation programs',
+      picture: 'statsv2/statImage5.png',
+      alt: 'Staff in front of FixNation building'
+    },
+    {
+      number: '35%',
+      subtitle: 'cats acquired as strays',
+      picture: 'statsv2/statImage6.png'
+    }
+  ];
+
+  let screenWidth: number;
+</script>
+
+<svelte:window bind:innerWidth={screenWidth} />
+
+<h1>
+  FixNation’s mission is to reduce the population of homeless cats by sterilizing as many as
+  possible while demonstrating the effectiveness of Trap-Neuter-Return and colony management for the
+  humane care of homeless cats.
+</h1>
+<div>
+  {#if screenWidth > 675}
+    <div class="slider">
+      {#each sections as section}
+        <section class="section">
+          <img src={section.picture} alt={section.alt} />
+          <span class="rectangle" />
+          <div class="text">
+            <div class="header">{section.number}</div>
+            <div class="subtitle">{section.subtitle}</div>
+          </div>
+        </section>
+      {/each}
+    </div>
+  {:else}{/if}
+</div>
+
+<style>
+  h1 {
+    font-family: 'Avenir', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    text-align: center;
+    font-size: 1rem;
+    margin: 1vw;
+    padding: 2rem;
+  }
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 1;
+  }
+
+  section {
+    padding: 1rem;
+    min-width: 23rem;
+    height: auto;
+    scroll-snap-align: start;
+    text-align: center;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
```suggestion
  }
  section.active {
      min-width: 16.65vw;
  }
```
Allows the carousel to center